src/conv.jl

```diff
@@ -62,8 +68,12 @@ function ∇conv_data!(dx::AbstractArray{T,3}, dy::AbstractArray{T,3},
 end
 
 conv!(y::AbstractArray{T,4}, x::AbstractArray{T,4}, w::AbstractArray{T,4};
-      pad = 0, stride = 1, dilation = 1) where T =
-  conv2d!(y, x, w, padding = pad, stride = stride, dilation = dilation)
+      pad = 0, stride = 1, dilation = 1, flipkernel=true) where T =
+  conv2d!(y, x, w, padding = pad, stride = stride, dilation = dilation, flipkernel=flipkernel)
+
+crosscor!(y::AbstractArray{T,4}, x::AbstractArray{T,4}, w::AbstractArray{T,4};
```

Comment:
Cross correlation should have `flipkernel=false`, or at least be different from `conv!`, no? I also think it'd be better to keep this generic to `AbstractArray`s and just forward whatever the arguments are to `conv!`.